tests/publiccloud/aistack_basic.pm

```diff
@@ -0,0 +1,145 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Basic aistack test
+
+# Summary: This test performs the following actions
+#  - Create a VM in EC2 using SLE-Micro-6-0-BYOS.x86_64-1.0.0-EC2-Build1.36.raw.xz
+#  - Install the required dependencies to install the aistack helm chart and containers
+#  - Test access to OpenWebUI and run integration tests with Ollama and MilvusDB
+# Maintainer: Yogalakshmi Arunachalam <yarunachalam@suse.com>
+#
+
+use Mojo::Base 'publiccloud::basetest';
+use testapi;
+use serial_terminal 'select_serial_terminal';
+use publiccloud::utils qw(is_byos registercloudguest register_openstack);
+use publiccloud::ssh_interactive 'select_host_console';
+use strict;
+use warnings;
+use utils;
+use publiccloud::utils;
+use File::Basename;
+use version_utils;
+use Data::Dumper;
+
+sub install_package {
```

Comment:
We have already a routine `install package` in `lib/package_utils.pm`, doing all needed in all cases.
Please check it if fits your needs and adapt your code accordingly.

Reply:
Taking different approach to install helm, docker, kubectl.. This routine may not be required.